setup.py

```diff
@@ -23,17 +23,17 @@
 Topic :: Utilities
 Intended Audience :: Developers
 """.strip().splitlines()
-KEYWORDS = "automation api testautomation testing qa robotframework"
+KEYWORDS = "automation api testautomation testing qa openapi"
```

Comment:
Isn't it good to leave robotframerk here? It will be searchable by bots/people in PyPi (interested in any libraries/tools related to Robot Framework). I think it helps to increase publicity and doesn't harm us.